api_tests/tests/reasonable_adjustment_flag/test_happy_cases.py

```diff
@@ -4,39 +4,74 @@
 import pytest
 import requests
 from assertpy import assert_that
+from api_tests.scripts.request_bank import BANK
 
 from api_tests.config_files import config
 from api_tests.tests.utils import Utils
 
 
 @pytest.mark.usefixtures("setup")
 class TestHappyCasesSuite:
-    """ A test suite to verify all the happy path oauth endpoints """
+    """ A test suite to verify all the happy path endpoints """
 
     @pytest.mark.happy_path
-    @pytest.mark.sandbox
```

Comment:
Removed these from running as the Sandbox expects a different NHS number to the one the tests now use. I am going to create a Jira ticket to fix the Sandbox